octobot_trading/exchanges/exchange_builder.py

```diff
@@ -94,7 +91,7 @@ def is_backtesting(self, backtesting_files):
         return self
 
     def is_sandboxed(self, sandboxed: bool):
-        self._is_exchange_manager_sandboxed = sandboxed
+        self.config[CONFIG_EXCHANGES][self.exchange_name][CONFIG_EXCHANGE_SANDBOXED] = sandboxed
```

Comment:
Not sure it's a good idea to edit the user config

Reply:
I agree, but it's a way of letting users set the sandbox mode from config and also forcing sandbox mode from bot params even if it's not set in user config.

Reply:
Right, in this case shouldn't we use something like:
```python
self._is_exchange_manager_sandboxed = self.config[CONFIG_EXCHANGES][self.exchange_name][CONFIG_EXCHANGE_SANDBOXED] or sandboxed
```